clang/include/clang/Sema/Sema.h

```diff
@@ -322,6 +322,9 @@ class SYCLIntegrationHeader {
   SYCLIntegrationHeader(DiagnosticsEngine &Diag, bool UnnamedLambdaSupport,
                         Sema &S);
 
+  /// Get c++ version to support version mismatch diagnostics
```

Comment:
I don't see a good reason for this to be a member function, it doesn't really seem to depend on the Sema or int-header.

Reply:
Updated and inlined in single instance where it's used